apps/site/lib/site_web/templates/trip_plan/_fare_calculator.html.eex

```diff
@@ -0,0 +1,61 @@
+
+<div class="h2 m-trip-plan-farecalc-title">Fare Calculator</div>
+
+<div class="m-trip-plan-farecalc-table">
+  <div class="m-trip-plan-farecalc-table-cell m-trip-plan-farecalc-subheader">One-Way Fare</div>
+  <div class="m-trip-plan-farecalc-table-cell m-trip-plan-results__itinerary-fare-title m-trip-plan-farecalc-header">Base</div>
+  <div class="m-trip-plan-farecalc-table-cell m-trip-plan-results__itinerary-fare-title m-trip-plan-farecalc-header">Recommended</div>
+  <div class="m-trip-plan-farecalc-table-cell m-trip-plan-results__itinerary-fare-title m-trip-plan-farecalc-header">Reduced</div>
+
+  <%= for {_fare_key, fare_values} <- @fares do %>
+
+    <% mode = fare_values.mode %>
+
+    <div class="m-trip-plan-farecalc-table-cell bold">
```

Comment:
This is a bit of a style question, but to me, it feels like the `bold` here and below aren't as semantically informative if they could be. Like maybe instead if we use a modifier, this one could be something like, `m-trip-plan-farecalc__table-cell--title`, and the one below might be, `m-trip-plan-farecalc__table-cell--total`.